TOC.md

```diff
@@ -360,16 +360,22 @@
         + [`inspection_result`](/system-tables/system-table-inspection-result.md)
         + [`inspection_summary`](/system-tables/system-table-inspection-summary.md)
   + UI
-    + TiDB Dashboard @施闻轩
-      + 访问
-      + 概况页面
-      + 集群信息页面
-      + 流量可视化页面
-      + SQL 语句分析页面
-      + 慢查询页面
-      + 集群诊断页面
-      + 日志搜索页面
-      + 实例性能分析页面
+    + TiDB Dashboard
```

Comment:
In https://github.com/pingcap/docs-cn/pull/3244 the TOC is requested to remove the author name. Should we do the same here?